shell/browser/atom_browser_client.h

```diff
@@ -15,13 +15,13 @@
 #include "base/synchronization/lock.h"
 #include "content/public/browser/content_browser_client.h"
 #include "content/public/browser/render_process_host_observer.h"
+#include "content/public/browser/web_contents.h"
```

Comment:
I'd prefer this be in the .cc file if possible, since we don't rely on the full class implementation in this .h file. In general, better to forward-declare in the header and only #include in the .cc, as it limits the blast radius of files that need to be recompiled when the header changes.